src/seba.jl

```diff
@@ -2,38 +2,38 @@
 # Implement SEBA (see https://web.maths.unsw.edu.au/~froyland/FRS18a.pdf)
 
 """
-    SEBA(V;[μ=0.99/sqrt(size(V,1)),tol=1e-14,maxiter=1000])
+    SEBA(V; [μ=0.99/sqrt(size(V,1)), tol=1e-14, maxiter=1000])
 
 Implements the SEBA algorithm (see https://web.maths.unsw.edu.au/~froyland/FRS18a.pdf).
 
 """
-function SEBA(Vin;μ=0.99/sqrt(size(Vin,1)),tol=1e-14,maxiter=5000)
-    n,nev = size(Vin)
+function SEBA(Vin; μ=0.99/sqrt(size(Vin, 1)), tol=1e-14, maxiter=5000)
+    n, nev = size(Vin)
     V = Matrix(qr(Vin).Q)
-    S = zeros(n,nev)
-    R = diagm(0 => ones(size(Vin,2)))#TODO: Allow this as optional argument?
+    S = zeros(n, nev)
+    R = diagm(0 => ones(size(Vin, 2)))#TODO: Allow this as optional argument?
     for i in 1:nev
         if maximum(V[:,i]) - minimum(V[:,i]) < 1e-14
-            V[:,i] .+= (rand(p) .- 0.5)*1e-12
```

Comment:
`p` was undefined. I have no idea how the docs got deployed...

Reply:
good catch, probably the if-condition didn't hold.